pkl-cli/src/test/kotlin/org/pkl/cli/CliEvaluatorTest.kt

```diff
@@ -1201,8 +1201,6 @@ result = someLib.x
     val err =
       assertThrows<CliException> { evalModuleThatImportsPackage(builtInCerts, packageServer.port) }
     assertThat(err)
-      // on some JDK11's this doesn't cause SSLHandshakeException but some other SSLException
-      // .hasMessageContaining("Error during SSL handshake with host `localhost`:")
```

Comment:
this line could be commented in instead of being removed

Reply:
Ah, I see. Will add!